salt/modules/mac_softwareupdate.py

```diff
@@ -44,7 +42,7 @@ def _get_available(recommended=False, restart=False, shut_down=False):
     cmd = ["softwareupdate", "--list"]
     out = salt.utils.mac_utils.execute_return_result(cmd)
 
-    if __grains__["osrelease_info"][1] >= 15:
+    if __grains__["osrelease_info"][0] > 10 or __grains__["osrelease_info"][1] >= 15:
```

Comment:
I'm not very close to Mac OS stuff. But if case conflicts with your description: 10.16 and 11. As I can google there's a Mac OS 10.15 that will pass this check. Could you please comment it?

Reply:
Gotcha-I believe it's correct.

So you would get for current Catalina releases `['10', '15', '6']` and for Big Sur `['11', '0', '0']`.
All previous years, the macOS versions have been 10.x; so e.g. `['10', '14', '6']` the prior year.

Reply:
I'm okay then. Thank you!